### ISSUE TITLE
build AWS AMI with EngHF: tags shows values for base build and not EngHF

### ISSUE THREAD

**Issue by amolari** [f5devcentral/f5-bigip-image-generator#12]
**Describe the bug**
Creating an image with a Hotfix (EngHF), without specifying any tags, some default tags will be automatically created. They do not reflect the build, as their values are from the image (base biuild) and not from the Hotfix.
Tags:
- version_sequence  
- version_build (which should have a different value as tag  version_basebuild)
(and maybe):
- version_changelist
- version_date
- version_edition

**To Reproduce**
Generate an AMI to AWS with a base image and an EngHF.

**Expected behavior**
Above Tags should have the value corresponding to the EngHF and not the base build

**Additional context**
Public Cloud AWS

**Comment by kmaloney** [comment]
Are you using the latest version of the Image Generator, v1.5 (released last week), or an earlier version?  

I tried to repro this in v1.5, and the tags seem to reflect the EngHF version file.  There were changes made in v1.5 to properly name EngHF images and check that the base image and EngHF image are compatible.  These changes required unpacking the EngHF version file from the EngHF image.  A side-effect of these changes is that the EngHF version file is used for tagging rather than the base image version file.

I confirmed that when I built an AWS image using Image Generator v1.4, that the behavior matches what you have described (tags reflect base image version file).
